hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/ContentGenerator.java

```diff
@@ -48,14 +49,37 @@
 
   private final byte[] buffer;
 
+  /**
+   * Issue Hsync after every write ( Cannot be used with Hflush ).
+   */
+  private final boolean hSync;
+
+  /**
+   * Issue Hflush after every write ( Cannot be used with Hsync ).
+   */
+  private final boolean hFlush;
+
   ContentGenerator(long keySize, int bufferSize) {
-    this(keySize, bufferSize, bufferSize);
+    this(keySize, bufferSize, bufferSize, false, false);
   }
 
   ContentGenerator(long keySize, int bufferSize, int copyBufferSize) {
+    this(keySize, bufferSize, copyBufferSize, false, false);
+  }
+
+  ContentGenerator(long keySize, int bufferSize, boolean hsync,
+                   boolean hflush) {
+    this(keySize, bufferSize, bufferSize, hsync, hflush);
+  }
+
+  ContentGenerator(long keySize, int bufferSize, int copyBufferSize,
```

Comment:
This is good. If you plan to add more parameters to ContentGenerator later, I'd suggest to make it a Builder: https://howtodoinjava.com/design-patterns/creational/builder-pattern-in-java/

Reply:
It's a great suggestion, but upon implementing it there will be other classes that will have to be changed as well, hence that is outside the scope of this JIRA. Instead I'll file another JIRA that will implement a builder class for the ContentGenerator class, this will help in accommodating more parameters in the future. 
Ill also post the link for the new JIRA here as well!

Reply:
Link to the filed JIRA :- https://issues.apache.org/jira/browse/HDDS-6455?filter=-1